src/test/java/de/wejago/mqtt2influx/service/JsonSubscriberTest.java

```diff
@@ -122,24 +121,34 @@ private void buildTestDevice1() {
 
     private void buildTestDevice2() {
         Map<String, String> deviceMappings = new HashMap<>();
-        deviceMappings.put("Total_in", "Total Consumption");
-        deviceMappings.put("Total_out", "Total Production");
-        deviceMappings.put("Power_curr", "Current Consumption");
-        deviceMappings.put("device_id", "Device ID");
+        deviceMappings.put("StatusSNS.SML.1_8_0", "Total Consumption");
+        deviceMappings.put("StatusSNS.SML.2_8_0", "Total Production");
+        deviceMappings.put("StatusSNS.SML.16_7_0", "Current Consumption");
+        deviceMappings.put("StatusSNS.SML.96_1_0", "Device ID");
         device2 = new Device();
         device2.setName("testName");
-        device2.setSensorId("device_id");
+        device2.setSensorId("StatusSNS.SML.96_1_0");
         device2.setOnlyMatch("1_8_0");
         device2.setMappings(deviceMappings);
     }
 
     private Point buildTestPoint() {
         return Point.measurement("sensor")
                 .time(Instant.now(), WritePrecision.MS)
-                .addTag("sensor_id", "0a01454d480000b22b25")
+                .addTag("sensor_id", "012345affecaffee")
                 .addTag("device_name", "testName")
                 .addField("Total Consumption", 695.38)
                 .addField("Total Production", 16.94)
                 .addField("Current Consumption", 1151D);
     }
+
+    private Point buildTestPoint2() {
+        return Point.measurement("sensor")
+                .time(Instant.now(), WritePrecision.MS)
+                .addTag("sensor_id", "012345affecaffee")
+                .addTag("device_name", "testName")
+                .addField("Total Consumption", 25273.16)
+                .addField("Total Production", 5.39)
+                .addField("Current Consumption", -9.59d);
```

Comment:
why not only `-9.59`

Reply:
It should ne a double, I test that explicitly.